src/calculateRentalCost.js

```diff
@@ -3,8 +3,18 @@
  *
  * @return {number}
  */
+
+const seven = 7;
+const three = 3;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= seven) {
+    return days * 40 - 50;
+  } else if (days >= three) {
+    return days * 40 - 20;
+  } else {
+    return 80;
```

Comment:
This return statement is incorrect. It should be `return days * DAILY_RENTAL_COST;`, where `DAILY_RENTAL_COST` is a constant equals to 40. It will calculate the total cost without any discounts.